services/api/service.go

```diff
@@ -1315,6 +1317,13 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 		return
 	}
 
+	// save execution payload to memcached as secondary backup to Redis
+	err = api.memcached.SaveExecutionPayload(payload.Slot(), payload.ProposerPubkey(), payload.BlockHash(), getPayloadResponse)
+	if err != nil {
+		// since execution payload would've successfully saved in Redis, an error is logged, but we don't return early
+		log.WithError(err).Warnf("failed saving execution payload in memcached [%v]", err)
```

Comment:
i wonder if it should abort here, like if saving to redis fails 🤔 thoughts?

Reply:
yeah haha I was flipping back and forth on this 😅 

I'll revert back to aborting here, I kinda like it better, makes it easier to troubleshoot when we fail fast

Reply:
done! please feel free to un-resolve if I missed anything